lib/utils/device/deploy.ts

```diff
@@ -536,7 +536,7 @@ export function generateTargetState(
 		// this composition entry (everything else in this
 		// function comes from the composition which doesn't
 		// change)
-		let contract = null;
+		let contract;
```

Comment:
This was due to the `"noImplicitAny": false,` for the js type checking having an effect on how inference works, but this new form is an improvement imo anyway